test/verify/check-machines

```diff
@@ -120,6 +120,13 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         b = self.browser
         b.click("tbody tr[data-row-id=vm-{0}] th".format(vmName)) # click on the row header
 
+    def waitVmRow(self, vmName, present=True):
+        b = self.browser
+        if present:
+            b.wait_present("tbody tr[data-row-id=vm-{0}] th".format(vmName))
```

Comment:
DRY - put this into a `selector` variable.

Reply:
Done